style.css

```diff
@@ -91,6 +90,283 @@
   margin-right: 10px;
 }
 
+.works-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  color: #1c1a19;
+  font-size: 40px;
+  text-align: center;
+  margin-top: 100px;
+}
+
+.works-title-line {
+  width: 90%;
+  height: 2px;
+  background-color: #c7c7c7;
+  margin: 50px auto;
+}
+
+.works-wrapper {
+  width: 100%;
+  display: grid;
+  gap: 50px;
+  grid-template-columns: 1fr;
+}
+
+.main-project-wrapper {
+  width: 325px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 100px;
+}
+
+.main-project-img {
+  width: 325px;
+  margin: 0 auto;
+}
+
+.main-project-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  font-size: 32px;
+  color: #091e42;
+  margin: 25px 0;
+}
+
+.main-project-description {
+  font-family: "Inter", sans-serif;
+  color: #091e42;
+}
+
+.main-project-tags {
+  display: flex;
+  margin: 10px 0;
+  list-style-type: none;
+}
+
+.main-project-tag {
+  font-family: "Inter", sans-serif;
+  color: #091e42;
+  margin: 10px 0;
+  margin-right: 10px;
+  font-size: 15px;
+  border: 1px solid #8993a4;
+  padding: 10px;
+}
+
+.see-project-btn {
+  background-color: #ff6b00;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  text-decoration: none;
+  padding: 10px 20px;
+  border: none;
+  margin: 10px 0;
+  cursor: pointer;
+}
+
+.see-project-btn:hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+}
+
+.see-project-btn:focus {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.see-project-btn:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+.other-project-wrapper {
+  width: 325px;
+  height: 300px;
+  margin: 0 auto;
+  background-image: url("images/other-projects.png");
+  display: flex;
+  align-items: flex-end;
+}
+
+.other-project-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  font-size: 32px;
+  color: white;
+  margin: 0 0 20px 20px;
+}
+
+.other-project-description {
+  font-family: "Inter", sans-serif;
+  color: white;
+  margin: 0 0 20px 20px;
+}
+
+.other-project-tags {
+  display: flex;
+  list-style-type: none;
+}
+
+.other-project-tag {
+  font-family: "Inter", sans-serif;
+  color: white;
+  margin: 0 10px 20px 20px;
+  font-size: 15px;
+  background-color: #5c5c69;
+  padding: 10px;
+}
```

Comment:
- [ ] Great job on creating the tags! :+1: However, to align with the Figma template, it is necessary to reduce the spacing between them and make the background transparent. This will not only make the tags look more visually appealing but also ensure that they are consistent with the overall design of the portfolio. By following these guidelines, the tags will be easier to read and will complement the design elements of the portfolio, ultimately enhancing the user experience.

**Your design**

![Screenshot from 2023-04-28 12-31-24](https://user-images.githubusercontent.com/105117832/235216051-7018df2c-b490-45c3-94c8-2e36f713048e.png)

**Figma**

![Screenshot from 2023-04-28 12-31-45](https://user-images.githubusercontent.com/105117832/235216082-206b0e6d-2a4e-43ab-a0fe-b92b93f5765f.png)